src/sonic-yang-models/yang-models/sonic-serial-console.yang

```diff
@@ -0,0 +1,37 @@
+//filename:  sonic-serial-console.yang
+module sonic-serial-console {
```

Comment:
>module [](http://example.com/codeflow?start=0&length=6)

@ganglyu Please help review the 2 Yang models.

Reply:
@i-davydenko 
Can you create separate PR for sonic yang models?

Reply:
@i-davydenko is there a PR for yang? can you please share?
also please add it to the HLD list of PRs for the feature

Reply:
This commnet is still active.